server/helpers/tags/getTagIds.js

```diff
@@ -12,6 +12,9 @@ const getTagIds = (req, res) => {
     // eslint-disable-next-line prefer-destructuring
     tags = req.body.tags;
   }
+  if (!tags.length) {
+    return Promise.resolve([]);
+  }
```

Comment:
This prevents assigning all tags to an article if no tags were selected while creating or updating the article